doc/nrf/software_maturity.rst

```diff
@@ -7,11 +7,25 @@ Software maturity levels
    :local:
    :depth: 2
 
-Following are the different software maturity levels shown for the different features:
+|NCS| supports its various features and components at different levels of software maturity.
+The tables at the end of this page summarize the maturity level for each feature and component supported in |NCS|.
+Three levels are used in the tables to classify the software maturity of each feature and component:
```

Comment:
```suggestion
The following categories are used in the tables to classify the software maturity of each feature and component:
```
To avoid using the number in the introduction of the list.